src/libraries/Common/src/Internal/Cryptography/Helpers.cs

```diff
@@ -30,14 +30,9 @@ internal static partial class Helpers
             return (byte[])(src.Clone());
         }
 
-        public static int GetPaddingSize(this SymmetricAlgorithm algorithm, CipherMode mode, int feedbackSizeBits)
+        public static int GetPaddingSize(this SymmetricAlgorithm algorithm, CipherMode mode, int feedbackSizeInBits)
         {
-            // CFB8 does not require any padding at all
-            // otherwise, it is always required to pad for block size
-            if (mode == CipherMode.CFB && feedbackSizeBits == 8)
-                return 1;
-
-            return algorithm.BlockSize / 8;
+            return (mode == CipherMode.CFB ? feedbackSizeInBits : algorithm.BlockSize) / 8;
```

Comment:
This "worked" but was technically wrong - the padding amount is always based on the feedback size. However, since the only other feedback size supported other than CFB8 was block-size CFB, (CFB64 for 3DES and CFB128 for AES) it all worked anyway. This just does it "right" without changing the behavior.